.travis.yml

```diff
@@ -2,11 +2,24 @@ language: c
 compiler:
   - clang
   - gcc
+
 before_install:
   - sudo apt-get update -qq
   - sudo apt-get install -qq libcunit1 libcunit1-doc libcunit1-dev
-before_script:
-  - cmake .
+
 script:
-  - make
-  - make test
+  - cd .. && cmake libuast && make && make test
```

Comment:
You can avoid any colliding moving to a completely different dir, just in case:

`mkdir -p ../libuast-build && cd ../libuast-build && cmake ../libuast && make && make test`